packages/gatsby-theme-docs/gatsby-node.js

```diff
@@ -496,6 +503,7 @@ async function createContentPages(
 
         break;
       }
+      // TODO is this case necessary?  The whole function is only querying content pages.
```

Comment:
Please review, I found this slightly weird case that should not actually ever happen.